portal/server/helpers/constructPayload.test.js

```diff
@@ -25,6 +25,27 @@ const payloadProperties = [
   'passwordConfirm',
 ];
 
+const mockBodyWithEmptyValues = {
```

Comment:
Can an assertion be added with currency and supplier currency object?

Reply:
Added tests for above.